client/src/TrackStore.ts

```diff
@@ -3,14 +3,15 @@ import BaseAnnotationStore from './BaseAnnotationStore';
 import { AnnotationId } from './BaseAnnotation';
 
 export default class TrackStore extends BaseAnnotationStore<Track> {
-  add(frame: number, defaultType: string, afterId?: AnnotationId) {
-    const track = new Track(this.getNewId(), {
+  add(frame: number, defaultType: string, afterId?: AnnotationId, overrideTrackId?: number) {
+    const newId = overrideTrackId ?? this.getNewId();
```

Comment:
right side `?? this.getNewId();` is dead code.  `trackStore.add` is only used in one place, and it provides the override.  Might as well just make it a required argument and drop support for `this.getNewId`?

Reply:
Thanks I think this led to me catching some other bugs.  The fact that everywhere we did a trackAdd we need to make sure if overrideTrackId is underfined it needs to be set to `cameraStore.getNewTrackId()` instead of the trackstore version.  Could have conflicting trackIds then.